fix: support debug for sdk generator and add doc (#530)
AmadeusITGroup/otter: packages/@ama-sdk/schematics/README.md

```

[Default swagger config](./schematics/java/client-core/swagger-codegen-java-client/config/swagger-codegen-config.json) will be used if `--swagger-config-path` is not provided.

### Debug the typescript generator

> The open api generator extracts a JSON data model with a lot of information describing the specification provided, and use this data model to feed the templates and generate the code.

Reviewer: ```suggestion
The OpenApi generator extracts an enhance JSON data model from the specification yaml and use this data model to feed the templates to generate the code.
```

Reviewer 2: ```suggestion
The OpenApi generator extracts an enhanced JSON data model from the specification YAML and uses this data model to feed the templates to generate the code.
```

If there is an issue with the files generated with the spec provided, the generator provides debugging features that log this data model.

You can use global property options to pass one or both of the following options:

> * debugModel - log the full json structure used to generate models

Reviewer: ```suggestion
* debugModel - logs the full json structure used to generate models
```
@sdo-1A ?

Reviewer 2: also is JSON better?

> * debugOperations - log the full json structure used to generate operations

Reviewer 2: ```suggestion
* debugOperations - logs the full JSON structure used to generate operations
```
